ui/src/common/ScrollUpButton.tsx

```diff
@@ -3,11 +3,36 @@ import KeyboardArrowUp from '@material-ui/icons/KeyboardArrowUp';
 import React, {Component} from 'react';
 
 class ScrollUpButton extends Component {
+    state = {
+        display: 'none',
+        opacity: 0,
+    };
+    componentDidMount() {
+        if (typeof window !== 'undefined') {
+            window.addEventListener('scroll', () => {
```

Comment:
The handler probably should be removed on https://reactjs.org/docs/react-component.html#componentwillunmount